lib/ex_doc/nodes.ex

```diff
@@ -20,6 +21,7 @@ defmodule ExDoc.ModuleNode do
   @type t :: %__MODULE__{
           id: nil | String.t(),
           title: nil | String.t(),
+          title_truncated: nil | String.t(),
```

Comment:
What about `nested_title`?

Reply:
I used `title_...` as the value "goes with" `title`, but I see how that might not be helpful. I'll change it 👍